src/move_to_front.rs

```diff
@@ -36,8 +36,137 @@ impl MoveToFrontDecoder {
     }
 }
 
+pub struct PositionalMTFEncoder {
+    selectors: Vec<u16>,
+    mtf_freq: Vec<usize>,
```

Comment:
Not a priority now but we could probably use something smaller than `usize` here